apps/judicial-system/web/messages/Core/tables.ts

```diff
@@ -52,4 +52,24 @@ export const tables = defineMessages({
     description:
       'Notaður sem texti sem segir til um hvort mál sé framlenging í málalista.',
   },
+  created: {
```

Comment:
I decided to keep these strings here for now for consistency, and we can then move them closer to a shared tables component when we've landed on how we want to refactor our tables